lambda/postSchedulePeerTeachers/Scheduler/src/main/java/db/Section.java

```diff
@@ -23,6 +26,10 @@ public Section(int courseId, String sectionNumber, int capacityPTs, int capacity
         currentTAs = 0;
     }
 
+    public int getSectionId() {
+        return sectionId;
+    }
+
     public int getCourseId() {
         return courseId;
     }
```

Comment:
Does Java do a thing that generates getters for you?

Reply:
Yes.

Reply:
lol